src/main/webapp/web.xml

```diff
@@ -3,12 +3,4 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
-    <servlet>
-        <servlet-name>index</servlet-name>
-        <servlet-class>mate.controller.IndexController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>index</servlet-name>
-        <url-pattern>/index</url-pattern>
-    </servlet-mapping>
 </web-app>
```

Comment:
also missed it at first review 🔍 
![image](https://user-images.githubusercontent.com/117590814/214777328-bfd5b21e-16fc-4cf7-8880-8473cf77fe02.png)
⬆️ from common mistakes 
make sure to apply changes everywhere in the code )